### PR TITLE
sql tuning: add EXPLAIN views 

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

This is a followup PR to https://github.com/pingcap/docs/pull/3858 - but they can be reviewed independently, and merge out of order.

This PR does not add this page to the TOC. I will only do that once all PRs are merged (this is one ~3-4 followup PRs).

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3862
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by ireneontheway** [comment]
As there is no file named import-example-data.md in docs-cn, I cannot fix the dead link. @TomShawn

File: explain-views.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: 使用视图解释语句
+summary: 了解 TiDB 中 `EXPLAIN` 语句返回的执行计划信息。
+---
+
+# 使用视图解释语句
+
+`EXPLAIN` 语句显示[视图](/views.md)引用的表和索引，而不是视图本身的名称。这是因为视图是一张虚拟表，本身并不存储任何数据。视图的定义会和语句的其余部分在 SQL 优化过程中进行合并。
+
+参考 [bikeshare 数据库示例](/import-example-data.md)可知以下两个查询执行方式类似：
```
Comment (TomShawn) [inline explain-views.md:10]: @ireneontheway We can remove the link.
```suggestion
以下两个示例查询的执行方式类似：
```

File: explain-views.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
+title: 使用视图解释语句
+summary: 了解 TiDB 中 `EXPLAIN` 语句返回的执行计划信息。
+---
+
+# 使用视图解释语句
+
+`EXPLAIN` 语句显示[视图](/views.md)引用的表和索引，而不是视图本身的名称。这是因为视图是一张虚拟表，本身并不存储任何数据。视图的定义会和语句的其余部分在 SQL 优化过程中进行合并。
+
+以下两个示例查询的执行方式类似：
```
Comment (TomShawn) [inline explain-views.md:10]:
```suggestion
参考 [bikeshare 数据库示例（英文）](https://docs.pingcap.com/tidb/stable/import-example-data)，以下两个示例查询的执行方式类似：
```

File: explain-views.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
+title: 使用视图解释语句
+summary: 了解 TiDB 中 `EXPLAIN` 语句返回的执行计划信息。
```
Comment (zz-jason) [inline explain-views.md:3]:
```suggestion
title: `EXPLAIN` 视图
summary: 了解 TiDB 中视图相关语句的执行计划。
```

File: explain-views.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+---
+
+# 使用视图解释语句
```
Comment (zz-jason) [inline explain-views.md:6]:
```suggestion
# `EXPLAIN` 视图
```

File: explain-views.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+`EXPLAIN` 语句显示[视图](/views.md)引用的表和索引，而不是视图本身的名称。这是因为视图是一张虚拟表，本身并不存储任何数据。视图的定义会和语句的其余部分在 SQL 优化过程中进行合并。
```
Comment (zz-jason) [inline explain-views.md:8]:
```suggestion
`EXPLAIN` 语句会显示[视图](/views.md)引用的表和索引，而不是视图本身的名称。这是因为视图是一张虚拟表，本身并不存储任何数据。视图的定义会和语句的其余部分在 SQL 优化过程中进行合并。
```

File: explain-views.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+| id                      | estRows     | task      | access object | operator info                             |
++-------------------------+-------------+-----------+---------------+-------------------------------------------+
+| TableReader_7           | 43.00       | root      |               | data:Selection_6                          |
+| └─Selection_6           | 43.00       | cop[tikv] |               | eq(bikeshare.trips.bike_number, "W00950") |
+|   └─TableFullScan_5     | 19117643.00 | cop[tikv] | table:trips   | keep order:false                          |
++-------------------------+-------------+-----------+---------------+-------------------------------------------+
+3 rows in set (0.00 sec)
+```
+
+在上述第一条语句中，索引用于满足视图定义，接着在 TiDB 读取表行时应用了 `bike_number = 'W00950'`。在第二条语句中，没有满足该语句的索引，并且使用了 `TableFullScan`。
```
Comment (zz-jason) [inline explain-views.md:75]: "索引用于满足视图定义" 读起来怪怪的，原文是什么来着？

Comment (TomShawn) [reply explain-views.md:75]: [`TiDB makes use of indexes that satisfy both the view definition and the statement itself. Consider the following composite index:`](https://github.com/pingcap/docs/pull/3862/files#diff-3bbd2a4727d2f2b033c2042ff3034332R77)

Comment (TomShawn) [reply explain-views.md:75]: @zz-jason 
```suggestion
执行以上第一条语句时使用了索引，满足视图定义，接着在 TiDB 读取行时应用了 `bike_number = 'W00950'` 条件。执行以上第二条语句时，不存在满足该语句的索引，因此使用了 `TableFullScan`。
```

Comment (TomShawn) [reply explain-views.md:75]: @zz-jason PTAL
```suggestion
执行以上第一条语句时使用了索引，满足视图定义，接着在 TiDB 读取行时应用了 `bike_number = 'W00950'` 条件。执行以上第二条语句时，不存在满足该语句的索引，因此使用了 `TableFullScan`。
```

File: explain-views.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
+---
+title: `EXPLAIN` 视图
```
Comment (TomShawn) [inline explain-views.md:2]:
```suggestion
title: EXPLAIN 视图
```

File: explain-views.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
+---
+title: EXPLAIN 视图
```
Comment (TomShawn) [inline explain-views.md:2]:
```suggestion
title: 使用 EXPLAIN 查看带有视图语句的执行计划
```

File: explain-views.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+summary: 了解 TiDB 中视图相关语句的执行计划。
+---
+
+# `EXPLAIN` 视图
```
Comment (TomShawn) [inline explain-views.md:6]:
```suggestion
# 使用 `EXPLAIN` 查看带有视图语句的执行计划
```

File: explain-views.md (hunk 8 above)
Comment (lilin90) [reply explain-views.md:2]: @TomShawn Has this been confirmed? The English title `EXPLAIN Statements Using Views` also sounds a little bit weird. It seems that EXPLAIN is used as a verb.

Comment (TomShawn) [reply explain-views.md:2]: + @SunRunAway PTAL whether my comment is correct, thanks!
+ Yes. `EXPLAIN Statements Using Views` is a bit weird and is modified in https://github.com/pingcap/docs/pull/4088.

File: explain-views.md (hunk 6 above)
Comment (zz-jason) [reply explain-views.md:75]: LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**